docs/_includes/navbar.html

```diff
@@ -49,15 +49,15 @@
 	  <ul class="nav navbar-nav">
 
 	    <li>
-	      <a href="index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
+	      <a href="/index.html" class="{% if page.url contains 'index.html' %}active{% endif %}">Documentation</a>
```

Comment:
Then we should try to change the way the docs are linked I think. Relative paths don't allow you to have any kind of directory structure for the html files. Maintaining a directory with all our html files will quickly become a big mess.